BookingsApi/BookingsApi.Domain/JusticeUser.cs

```diff
@@ -14,7 +14,18 @@ public JusticeUser()
             VhoWorkHours = new List<VhoWorkHours>();
             Allocations = new List<Allocation>();
         }
-        
+
+
+        public JusticeUser(string firstName, string lastname, string contactEmail, string username, UserRole userRole) : this()
+        {
+            FirstName = firstName;
+            Lastname = lastname;
+            ContactEmail = contactEmail;
+            Username = username;
+            UserRoleId = userRole.Id;
+            CreatedDate = DateTime.UtcNow;
```

Comment:
The created date already gets set in the parent TrackableEntity() constructor I think

Reply:
You're right, thanks, I will remove that line in the JusticeUser ctor